src/main/java/org/jitsi/videobridge/cc/BitrateController.java

```diff
@@ -1182,6 +1187,54 @@ public boolean transformRtp(@NotNull PacketInfo packetInfo)
         }
     }
 
+    /**
+     * Subscribe to {@link BitrateController}'s events.
+     * @param listener event listener
+     */
+    public void subscribe(BitrateControllerListener listener) {
+        listeners.add(listener);
+    }
+
+    /**
+     * Unsubscribe from {@link BitrateController}'s events.
+     * @param listener event listener
+     */
+    public void unsubscribe(BitrateControllerListener listener) {
+        listeners.remove(listener);
+    }
+
+    private void notifyForwardedEndpointsChanged(
+        Collection<String> forwardedEndpoints,
+        Collection<String> endpointsEnteringLastN,
+        Collection<String> conferenceEndpoints) {
+        for (BitrateControllerListener listener : listeners) {
+            listener.onForwardedEndpointsChanged(
+                forwardedEndpoints, endpointsEnteringLastN, conferenceEndpoints);
+        }
+    }
+    /**
+     * Interface for listeners of events generated
+     * by {@code BitrateController}
+     */
+    public interface BitrateControllerListener {
```

Comment:
Can you rename this to `EventHandler`? Then it gets referred to as `BitrateController.EventHandler`